### ISSUE TITLE
Onderwerp NWWZ vlgs gezinssituatie % 2018 en 2019

### ISSUE THREAD

**Issue by liesbetvandriessche** [provinciesincijfers/bugsinjive#74]
De % NWWZ volgens gezinssituatie (t.o.v. de inw. 18-64 jaar van die gezinssituatie) van 2018 en 2019 geven allen 100% aan. Voor de jaren 2014 tot en met 2017 kloppen de percentages.
ter info: https://kortrijk.incijfers.be//jive?workspace_guid=dc1d51fd-bdf0-44c3-bf3b-150335572722

het gaat om de onderwerpcodes:
dna_vp1309_ba_pos_eo_gnminkind
dna_vp1309_ba_pos_eo_minkind
dna_vp1309_ba_pos_partner_gnminkind
dna_vp1309_ba_pos_partner_minkind
dna_vp1309_ba_pos_kind_ouder
dna_vp1309_ba_pos_andere

**Comment by joostschouppe** [comment]
Inderdaad; dit zit bij ons nog steeds fout: https://provincies.incijfers.be/admin/jive?var=vp1309_ba_pos_eo_gnminkind
Ik geef het door aan @VickyVB
